### PR TITLE
add indixes for versions

### PR DESCRIPTION
**Pull request by casiodk**

### CODE REVIEW

File: priv/repo/migrations/20181120130300_add_more_indicies_to_versions.exs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+defmodule VersionWarehouse.Repo.Migrations.AddMoreIndiciesToVersions do
+  use Ecto.Migration
+
+  def change do
+    create index(
```
Comment (larssg) [inline priv/repo/migrations/20181120130300_add_more_indicies_to_versions.exs]: We need to make sure this doesn't block

File: priv/repo/migrations/20181120130300_add_more_indicies_to_versions.exs [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+      :versions,
+      [:item_type, :event, :created_at]
```
Comment (dipth) [inline priv/repo/migrations/20181120130300_add_more_indicies_to_versions.exs]:
```suggestion
      [:item_type, :event, :created_at],
      concurrently: true
```

**Comment by andreaseriksson** [comment]
@larssg @dipth We don't need to add `@disable_ddl_transaction true`
https://hexdocs.pm/ecto/1.0.1/Ecto.Migration.html

**Comment by casiodk** [comment]
@dipth @casiodk i accepted your changes and im merging now

**Pull request merged**